i18n.md

```diff
@@ -0,0 +1,214 @@
+```
+
+These forms are not acceptable:
+
+```
+_("%s (All %s)" % [@ems_cluster.name, title_for_hosts]) # the name of the placeholder can provide vital information to the translator
+_("No %s were selected to move up") % "fields"          # use named placeholder even in the case of a single placeholder
+```
+
+* do not use variables inside gettext strings as these will not be extracted and placed in the gettext catalog
+
+Incorrect:
+
+```
+text = "Some random text"
+_(text)
+```
+
+Correct:
+
+```
+_("Some random text")
+```
+
+### Marking gettext strings in UI
+
+To be able to see strings which pass through gettext (i.e. are translatable), you have to add the following to the servers advanced settings:
+
+```
+ui:
+  mark_translated_strings: true
+```
+and restart the ManageIQ application. With these settings on, all strings passing through gettext will be marked with `»` and `«` markers:
+
+```
+»webui text that went through some of the gettext routines«
+```
+
+
+## Translating ManageIQ
+
+To contribute with translation to ManageIQ, create an account at [zanata](translate.zanata.org)
+and navigate to [ManageIQ project page](https://translate.zanata.org/zanata/project/view/manageiq)
+
+### Translator notes
+
+* How do I translate keys in the form of `Hardware|Usage`? What do they mean?
+`Hardware|Usage` means Usage in namespace Hardware. This is the way we translate model attributes for ActiveRecord models for example. You do not have to translate "Hardware", just translate "Usage".
+
+* what does the key `locale_name` mean?
+`local_name` meens name of the given language in the language itself. Such as "Deutsch" for German or "Česky" for Czech. Make sure to provide the value for this key, without it the language/locale cannot be presented in the UI.
+
+
+## Updating translations (developers)
+
+The workflow for updating translations is:
+
+* Update gettext catalogs in ManageIQ git repository. This is done to make sure the gettext catalog contains up to date (i.e. current) strings for translators. This part entails several steps.
+
+First, comment out `config.eager_load_paths = []` in `config/application.rb`
+
+Next, update `config/model_attributes.rb` to reflect current status. From the root of ManageIQ checkout, run the following:
+
+```
+$ bundle exec rake gettext:store_model_attributes
+```
+
+Next, update gettext catalog by running:
+
+```
+$ bundle exec rake gettext:find
+```
+
+Next, revert unneeded changes and clean up unnecessary files:
+
+```
+$ git reset config/application.rb
+$ git clean -f
+```
+
+Now, commit and push changed files into git (branch, pull request, etc.). Make sure no other files than the following list is commited:
+
+```
+config/model_attributes.rb
+config/locales/manageiq.pot
+config/locales/*/*.po
+```
+
+* Upload the catalog created in previous step to [zanata](http://translate.zanata.org). You need to be maintainer of ManageIQ project at [zanata](http://translate.zanata.org)
```

Comment:
@mzazrivec Note that I am working on making it so we don't have to make every contributor a maintainer.  The next version of zanata will have Team-specific languages groups, and we can make people within those groups maintainers, reviewers, or just translators.